backend/TraderX/src/main/java/cmpe451/group6/rest/equipment/configuration/EquipmentConfig.java

```diff
@@ -0,0 +1,23 @@
+package cmpe451.group6.rest.equipment.configuration;
+
+import java.text.DateFormat;
+import java.text.SimpleDateFormat;
+
+public class EquipmentConfig {
+
+    public static final DateFormat df = new SimpleDateFormat("yyyy-MM-dd");
```

Comment:
Is this the format that we want?

Reply:
This is the format the third party API service uses. Date value is stored and send back to client as timestamps